core/src/i18n/en.yaml

```diff
@@ -125,6 +125,11 @@ namespaces:
     create: Select this option to disable Istio to mediate all communication between the Pods in your Namespace.
     memory-examples: Use plain value in bytes (128974848) or suffix equivalents (like 129e6, 129M, 123Mi).
     total-memory-limit: Define constraints that limit total memory consumption in your Namespace.
+navigation:
```

Comment:
maybe a `profile`/`profile-dropdown` instead of `navigation`? that's how the dropdown is called in the sap/luigi world :)